### DISCUSSION TITLE
RocksDB on Mobile?

### DISCUSSION THREAD

**Discussion by davidhmays** [cozodb/cozo#193] [Q&A]
Reading the documentation, it mentions the difficulty of compiling RocksDB for mobile, but I didn't quite understand if CozoDB is able to use RocksDB on mobile, having somehow gotten past thes difficulties, or if SqLite should be used instead if on mobile? (The benchmarks sure show SqLite being way slower, so was hoping the RocksDB would work fine.)

Also it sounds like there is a drop-in replacement available for RocksDB that is supposedly even faster called [Speedb](https://www.speedb.io/) maybe could be worth looking into, and the site seems to mention mobile as a use case.

**Comment by simonwh** [comment]
Very interested in this as well, currently evaluating Cozo for use in React Native for a local-first app. Also looking at incremental sync, possible with sqlite through [cr-sqlite](https://github.com/vlcn-io/cr-sqlite) or alternatives. 

Eager to hear from the community on any learnings on mobile!
